rosa_hcp/rosa-hcp-creating-cluster-with-aws-kms-key.adoc

```diff
@@ -16,7 +16,7 @@ Create a {product-title} (ROSA) with a {hcp} (HCP) cluster  using a custom AWS K
 
 To create a {hcp-title} cluster, you must have the following items:
 
-* A configured virtual private cloud (VPC)
+* A configured virtual private cloud (VPC) that is not shared between multiple AWS accounts
```

Comment:
I worry that this additional text may cause unnecessary confusion. I suspect just adding the admonition snippet should be sufficient.

Reply:
Is that because it's too wordy? Would it be better to say "that is not a shared VPC"?

Reply:
Removed for now, we can add in future if needed.

Reply:
A "Shared VPC" is honestly a pretty _unusual_ configuration in the first place, so if we add these "weird callouts" by changing headers and what not, I worry it will end up causing _more_ confusion than it helps